doc/tutorials/SIR-X.ipynb

```diff
@@ -306,6 +322,78 @@
     "print(\"Quarantine probability =  %.2f, Maier and Brockmann = %.2f\" % (g_sirx.q_prob, 0.51))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Make predictions using `model.predict`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Make predictions and visualize\n",
+    "# Obtain the results 14 days after the train data ends\n",
+    "sirx_pred = g_sirx.predict(14)\n",
+    "print('T n_S \\t   n_I \\tn_R \\tn_X')\n",
+    "for i in sirx_pred:\n",
+    "    print(*i.astype(int))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Prepare date time index to plot predictions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    " # Import datetime module from the standard library\n",
+    "import datetime\n",
+    "# Obtain the last day from the data used to train the model\n",
+    "last_time = ts_fit.index[-1] \n",
+    "# Create a date time range based on the number of rows of the prediction\n",
+    "numdays = sirx_pred.shape[0] \n",
+    "day_zero = datetime.datetime(last_time.year,last_time.month,last_time.day)\n",
+    "date_list = [day_zero + datetime.timedelta(days=x) for x in range(numdays)]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Plot predictions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Extract figure and axes\n",
+    "fig, ax = plt.subplots(figsize = [5,5])\n",
+    "# Create core plot attributes\n",
+    "plt.plot(date_list, sirx_pred[:,4], color = 'red', linewidth =2)\n",
```

Comment:
Maybe we should check if there is an easy way to lint `.ipynb` files 🤔

Reply:
Let me check. I am going to review the notebooks and try to keep them compliant to our lint practices